qemu-lite/qemu-lite.dsc-template

```diff
@@ -1,13 +1,13 @@
 Format: 3.0 (quilt)
 Source: qemu-lite
-Version: 2.7.1
+Version: @VERSION@-@RELEASE@
```

Comment:
@gorozco1 I think this line should also have the commit ID

Reply:
@erick0z updated.